runbook.md

```diff
@@ -182,10 +182,17 @@ This page collects this repositories alerts and begins the process of describing
 ### Group Name: "kubernetes-system"
 
 ##### Alert Name: "KubeNodeNotReady"
-+ *Message*: `{{ $labels.node }} has been unready for more than 15 minutes."`
++ *Message*: `{{ $labels.node }} has been unready for more than 15 minutes.`
 + *Severity*: warning
 + *Runbook*: [Link](https://runbooks.prometheus-operator.dev/runbooks/kubernetes/kubenodenotready/)
 
+##### Alert Name: "KubeNodePressure"
++ *Message*: `{{ $labels.node }} has active Condition {{ $labels.condition }}. This is caused by resource usage exceeding eviction thresholds.`
++ *Severity*: info
++ *Runbook*: [Link](https://runbooks.prometheus-operator.dev/runbooks/kubernetes/kubenodepressure/)
+
+# If soft thresholds are crossed, pods will be evicted respecting TerminationGracePeriod. If Hard thresholds are crossed grace period will be ignored.
```

Comment:
The CI check was just complaining about some whitespace... I fixed it in 0478f9b.

Reply:
ah thank you!